lib/instrumental.js

```diff
@@ -233,6 +237,21 @@ exports.init = function instrumental_init(startup_time, config, events) {
       recordCounterRates = config.instrumental.recordCounterRates;
     }
 
+    // do no filtering by default
+    if(typeof(config.instrumental.metricFilters) === 'undefined'){
+      metricFilters = [];
+    } else {
+      metricFilters = config.instrumental.metricFilters;
+      filterMetric = function(metric) {
+        metricName = metric.split(" ")[1]
+        passedFilter = true;
+        metricFilters.forEach(function(filter) {
+          passedFilter = metricName.match(filter);
+        });
+        return passedFilter
```

Comment:
What if, instead of `forEach`, we use [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)? Maybe like this...

```
filterMetric = function(metric){
  metricName = metric.split(" ")[1];
  return metricFilters.some(function(filter){
    return metricName.match(filter);
  };
}
```

...or, if we're okay with arrow functions:

```
filterMetric = function(metric){
  metricName = metric.split(" ")[1];
  return metricFilters.some((filter)=> metricName.match(filter));
  };
}
```